handlers/s3client.go

```diff
@@ -33,13 +35,14 @@ func CreateNewAwsClient() (*AwsClient, error) {
 	return client, nil
 }
 
-// StreamObjectFromS3 downloads object from s3
-func StreamObjectFromS3(client *AwsClient, bucket string, key string) ([]byte, error) {
+// GetChunkDataFromS3 downloads chuck data from s3
```

Comment:
typo: ~`chuck`~ -> `chunk`

Reply:
Fixed in f6d90ae451c37d43f7ee3f6e7fd753ccdb1f6580